COPYRIGHT.txt

```diff
@@ -86,6 +86,11 @@ Comment: The Android Open Source Project
 Copyright: 2002, Google Inc.
 License: Apache-2.0
 
+Files: ./platform/android/java/lib/src/org/godotengine/godot/utils/ProcessPhoenix.java
+Comment: ProcessPhoenix (commit #12cb27c2cc9c3fc555e97f2db89e571667de82c4)
```

Comment:
I would put the commit in the `.java` file itself. This comment is used as title for the thirdparty licenses in the About dialog, such a long name will look pretty bad.

Reply:
Something like this:
```diff
diff --git a/COPYRIGHT.txt b/COPYRIGHT.txt
index 17ddcadfe0..64d9a93b9f 100644
--- a/COPYRIGHT.txt
+++ b/COPYRIGHT.txt
@@ -87,7 +87,7 @@ Copyright: 2002, Google Inc.
 License: Apache-2.0
 
 Files: ./platform/android/java/lib/src/org/godotengine/godot/utils/ProcessPhoenix.java
-Comment: ProcessPhoenix (commit #12cb27c2cc9c3fc555e97f2db89e571667de82c4)
+Comment: ProcessPhoenix
 Copyright: 2015, Jake Wharton
 License: Apache-2.0
 
diff --git a/platform/android/java/lib/src/org/godotengine/godot/utils/ProcessPhoenix.java b/platform/android/java/lib/src/org/godotengine/godot/utils/ProcessPhoenix.java
index 900cf254f0..2cc37b627a 100644
--- a/platform/android/java/lib/src/org/godotengine/godot/utils/ProcessPhoenix.java
+++ b/platform/android/java/lib/src/org/godotengine/godot/utils/ProcessPhoenix.java
@@ -1,5 +1,10 @@
 // clang-format off
 
+/* Third-party library.
+ * Upstream: https://github.com/JakeWharton/ProcessPhoenix
+ * Commit: 12cb27c2cc9c3fc555e97f2db89e571667de82c4
+ */
+
 /*
  * Copyright (C) 2014 Jake Wharton
  *
```

Reply:
Done!